### PR TITLE
Fix the missing part of ks-jenkins configmap

### PR DESCRIPTION
**Pull request by LinuxSuRen**
/area devops

Fixes https://github.com/kubesphere/ks-jenkins/issues/11

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
fixes https://github.com/kubesphere/kubesphere/issues/3170

**Comment by shaowenchen** [comment]
/hold

    "playbook": "/kubesphere/playbooks/devops.yaml",
    "playbook_uuid": "44747a39-f834-4900-aee0-e8a4e66bb0de",
    "remote_addr": "127.0.0.1",
    "res": {
      "_ansible_no_log": false,
      "changed": true,
      "cmd": "/usr/local/bin/helm upgrade --install ks-jenkins /kubesphere/kubesphere/jenkins/jenkins -f /kubesphere/kubesphere/jenkins/custom-values-jenkins.yaml --namespace kubesphere-devops-system\n",
      "delta": "0:00:00.401062",
      "end": "2020-12-02 10:01:22.643100",
      "invocation": {
        "module_args": {
          "_raw_params": "/usr/local/bin/helm upgrade --install ks-jenkins /kubesphere/kubesphere/jenkins/jenkins -f /kubesphere/kubesphere/jenkins/custom-values-jenkins.yaml --namespace kubesphere-devops-system\n",
          "_uses_shell": true,
          "argv": null,
          "chdir": null,
          "creates": null,
          "executable": null,
          "removes": null,
          "stdin": null,
          "stdin_add_newline": true,
          "strip_empty_ends": true,
          "warn": true
        }
      },
      "msg": "non-zero return code",
      "rc": 1,
      "start": "2020-12-02 10:01:22.242038",
      "stderr": "coalesce.go:199: warning: destination for InitScripts is a table. Ignoring non-table value <nil>\nError: parse error at (jenkins/templates/config.yaml:227): unexpected {{else}}",
      "stderr_lines": [
        "coalesce.go:199: warning: destination for InitScripts is a table. Ignoring non-table value <nil>",
        "Error: parse error at (jenkins/templates/config.yaml:227): unexpected {{else}}"
      ],
      "stdout": "Release \"ks-jenkins\" does not exist. Installing it now.",
      "stdout_lines": [
        "Release \"ks-jenkins\" does not exist. Installing it now."
      ]
    },
    "role": "ks-devops/jenkins",
    "start": "2020-12-02T02:01:21.905612",
    "task": "Jenkins | Deploy Jenkins",
    "task_action": "command",
    "task_args": "",
    "task_path": "/kubesphere/installer/roles/ks-devops/jenkins/tasks/main.yaml:66",
    "task_uuid": "72fe2e39-0ccd-e38a-d109-000000000039",
    "uuid": "75823a23-ab03-4463-b841-5babb07dcf13"
  },
  "parent_uuid": "72fe2e39-0ccd-e38a-d109-000000000039",
  "pid": 2960,
  "runner_ident": "devops",
  "start_line": 105,
  "stdout": "fatal: [localhost]: FAILED! => {\"changed\": true, \"cmd\": \"/usr/local/bin/helm upgrade --install ks-jenkins /kubesphere/kubesphere/jenkins/jenkins -f /kubesphere/kubesphere/jenkins/custom-values-jenkins.yaml --namespace kubesphere-devops-system\\n\", \"delta\": \"0:00:00.401062\", \"end\": \"2020-12-02 10:01:22.643100\", \"msg\": \"non-zero return code\", \"rc\": 1, \"start\": \"2020-12-02 10:01:22.242038\", \"stderr\": \"coalesce.go:199: warning: destination for InitScripts is a table. Ignoring non-table value <nil>\\nError: parse error at (jenkins/templates/config.yaml:227): unexpected {{else}}\", \"stderr_lines\": [\"coalesce.go:199: warning: destination for InitScripts is a table. Ignoring non-table value <nil>\", \"Error: parse error at (jenkins/templates/config.yaml:227): unexpected {{else}}\"], \"stdout\": \"Release \\\"ks-jenkins\\\" does not exist. Installing it now.\", \"stdout_lines\": [\"Release \\\"ks-jenkins\\\" does not exist. Installing it now.\"]}",
  "uuid": "75823a23-ab03-4463-b841-5babb07dcf13"
}
******************************************************************************************************************************************************

File: roles/ks-devops/jenkins/files/jenkins/jenkins/templates/config.yaml [hunk 1]
```diff
@@ -220,4 +220,10 @@ data:
       cp -n /var/jenkins_jobs/$job /var/jenkins_home/jobs/$job/config.xml
     done
 {{- end }}
-{{- end }}
+{{- range $key, $val := .Values.Master.InitScripts }}
+  init{{ $key }}.groovy: |-
+{{ $val | indent 4 }}
+{{- end }}	{{- end }}
+{{ else }}
```
Comment (LinuxSuRen) [inline roles/ks-devops/jenkins/files/jenkins/jenkins/templates/config.yaml:227]: I just let keep the same as before. https://github.com/kubesphere/ks-installer/blob/release-3.0/roles/ks-devops/jenkins/files/jenkins/jenkins/templates/config.yaml#L245

Anyway, I'll test the 'else' case.

Comment (LinuxSuRen) [reply roles/ks-devops/jenkins/files/jenkins/jenkins/templates/config.yaml]: fixed this via https://github.com/kubesphere/ks-installer/pull/1174/commits/c4a0cea89080c5489ec78637df55f8acc8b02242

**Pull request merged**